DayrSuryan/DSEthiop7.xml

```diff
@@ -84,9 +84,9 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                      <textLang mainLang="gez"/>
                      <incipit xml:lang="gez"><hi rend="rubric">ጸሎተ፡</hi> ሃይማኖት፡ ነአምን፡ ፩አምላክ፡ እ<hi rend="ligature">ግዚ</hi>አብሔር፡ አብ፡ አኃዜ፡ ኵሉ፡ ገባሬ፡ ሰማያት፡
                         ወምድር፡ ዘያስተርአ፡ ወዘ<pb n="50v"/>ኢያስተርኢ፡ ወነአምን፡ በ፩ እግ<gap reason="illegible"/>እ፡ ኢየሱስ፡ ክርስቶስ፡
-                        ወ<del rend="expunctuated"/>ልደ፡ አ<gap reason="illegible"/>ዋሕድ፡ ዘህልው፡
+                        ወልደ፡ አ<supplied resp="DN" reason="undefined">ብ፡</supplied> ዋሕድ፡ ዘህልው፡
```

Comment:
not expuntuated in the manuscript?

Reply:
Ok, it is a border case. I would not be able to say which is the letter that is cancelled. It was just a stroke, maybe just casual. There is no sense in indicating smth was corrected, if we cannot say what exactly was wrong.  It distracts.

Reply:
Good!